api.js

```diff
@@ -38,8 +44,11 @@ export function postComment(article_id, comment) {
 }
 
 export function deleteComment(comment_id) {
-    console.log(comment_id)
     return axios
     .delete(`https://h-nc-news-project.onrender.com/api/comments/${comment_id}`)
 }
 
+export function getTopics() {
+    return axios
+    .get("https://h-nc-news-project.onrender.com/api/topics")
```

Comment:
Higher up you're returning res.data.articles, whereas here the whole response will be returned into the component. Please make this consistent.